app/views/devise/_links.erb

```diff
@@ -0,0 +1,3 @@
+<% ActiveSupport::Deprecation.warn "Rendering partials devise/_links.erb is deprecated" \
```

Comment:
It was generated by ```devise-i18n-views``` I would keep it, but put a comment telling that is here just because was autogenerated and is not used.